Classes/Em/AjaxController.php

```diff
@@ -86,16 +86,15 @@ public function analyze(ServerRequestInterface $request, ResponseInterface $resp
 
                 // Populate the possible mapping configuration file names
                 $mappingFileNames = $extractionService->getPotentialMappingFiles($file);
-                $settings = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['extractor']);
+                $settings = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['extractor'];
```

Comment:
I would prefer to use ExtensionConfiguration class instead

Reply:
Switched to ExtensionSettingsTrait